README.md

```diff
@@ -37,6 +37,9 @@ fingerprint/biometric method.
 certain action or to see a given page. E.g. admin users are authorised to create
 new users on the website.
 
+**Declarations** - in JavaScript declarations are `var`, `const` and `let` for
+the creation of variables.
```

Comment:
"for the creation of variables" ... > "variables, constants and lexically scoped variables" 😉